.github/workflows/blank.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Run a one-line script
+      run: echo Hello, world!
+    - name: Run a multi-line script
+      run: |
+        echo Add other actions to build,
+        echo test, and deploy your project.
```

Comment:
Hey @parasgathani7143 the changes reflected in your PR is this .yml file. which is not required.
1. Please undo this change
2. Add the images in the screenshots folder then push the changes
If you still face troubles refer to some tutorial on how to commit your changes and create a Pull request.
Looking forward to improvements in this PR

Reply:
@vanshikaarora, don't forget to mark the GCI task as "More Work Needed" to reset the clock!